alibi/models/pytorch/tests/test_model.py

```diff
@@ -0,0 +1,618 @@
+import contextlib
+import os
+import tempfile
+from copy import deepcopy
+from typing import List
+
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data import DataLoader, TensorDataset
+
+from alibi.models.pytorch.metrics import AccuracyMetric, LossContainer
+from alibi.models.pytorch.model import Model
+
+
+class UnimodalModel(Model):
+    """ Simple uni-modal output model. """
+
+    def __init__(self, input_dim: int, output_dim: int):
+        super().__init__()
+        self.fc1 = nn.Linear(input_dim, output_dim, bias=False)
+        self.to(self.device)
```

Comment:
What's the reason for doing this here (contrasting with other tests where `cpu()` was called explicitly after model instance creation).

Reply:
See comment above. Probably can be deleted and was just added for some consistency. Guess I will just leave it there?